recipes/sentry-native/all/conanfile.py

```diff
@@ -16,7 +16,7 @@ class SentryNativeConan(ConanFile):
     topics = ("conan", "breakpad", "crashpad",
               "error-reporting", "crash-reporting")
     exports_sources = ["CMakeLists.txt", "patches/*"]
-    generators = "cmake", "cmake_find_package"
+    generators = "cmake", "cmake_find_package", "pkg_config"
     settings = "os", "arch", "compiler", "build_type"
```

Comment:
Where is the .pc used?
You also need to add pkg_conf as build requirement then.

Reply:
https://github.com/getsentry/sentry-native/blob/master/CMakeLists.txt#L441-L442

The builds passed so pkgconfig is installed on the build machine. Should I add it anyways? Why don't we do this with other tools like cmake?

Reply:
All recipes here expect cmake installed, so it's a mandatory requirement. If you haven't installed it, you need to add it to your profile.
pkg-config is optional.

Reply:
I see. I'll add it as a build_req then.